docs/governance/community-policies/definition-of-done.md

```diff
@@ -0,0 +1,14 @@
+# Definition of Done (DoD) for a Service
```

Comment:
Is this intended to include the end-to-end delivery of a service, both taxonomy definition and control definitions?

Reply:
No this was written with only the taxonomy in mind. Happy if someone can extend this to cover control definition